src/lib/services/encryption-service.ts

```
@@ -0,0 +1,72 @@
+	// Derive key from master key and salt
+	const key = crypto.pbkdf2Sync(secretKey, salt, PBKDF2_ITERATIONS, KEY_LENGTH, 'sha256');
+
+	const cipher = crypto.createCipheriv('aes-256-gcm', key, iv);
+
+	let encrypted = cipher.update(JSON.stringify(data), 'utf8', 'hex');
+	encrypted += cipher.final('hex');
+
+	const authTag = cipher.getAuthTag();
+
+	// Format: salt:iv:authTag:encrypted
+	return salt.toString('hex') + ':' + iv.toString('hex') + ':' + authTag.toString('hex') + ':' + encrypted;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)